install.sh

```diff
@@ -77,6 +77,8 @@ HELM_ARCHIVE=helm-${HELM_VERSION}-linux-amd64.tar.gz
 HELM_URL=https://get.helm.sh/$HELM_ARCHIVE
 DEFAULT_GATEWAY_PRIVATE_IP=192.168.1.1
 NAMESPACE_DEFAULT="liqo"
+LIQO_VERSION_DEFAULT="latest"
```

Comment:
Should the default be *latest* or *master*?

Reply:
It temporarily latest, it will be replaced by a specific version when the first one will be available.